src/components/charts/galaxySelector/index.jsx

```diff
@@ -414,14 +433,17 @@ class GalaxySelector extends React.PureComponent {
               alt={image.altText}
             />
           ) : (
-            <Blinker
-              images={images}
-              activeId={activeImageId}
-              playing={playing}
-              handleStartStop={this.startStopBlink}
-              handleNext={this.onNextBlink}
-              handlePrevious={this.onPreviousBlink}
-            />
+            <>
+              <Blinker
+                images={images}
+                activeId={activeImageId}
+                playing={playing}
+                handleStartStop={this.startStopBlink}
+                handleNext={this.onNextBlink}
+                handlePrevious={this.onPreviousBlink}
+              />
+              <ElapsedTime {...this.getAlertDaysAndHours(activeImageId)} />
```

Comment:
wrap in `{alerts && <ElapsedTime ... />}` since we wouldn't wanna render this counter if there's nothing to count.  Plus, then you don't need to check to see if alerts are present in your `getAlertDaysAndHours` method